src/Api/Events.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Fungku\HubSpot\Api;
+
+class Events extends Api
+{
+    protected $baseUrl = "http://track.hubspot.com/v1";
+
+    /**
+     * Add a custom event.
+     *
+     * This only works for enterprise accounts.
+     *
+     * @param  string $hubId Your HubSpot portal ID ("Hub ID"). You can find your Hub ID in the
+     *                       footer of the HubSpot UI, or in the URL. For example, in this URL:
+     *                       "https://app.hubspot.com/reports/56043/events/" your Hub ID is "56043".
+     * @param  string $eventId
+     * @param  string $contactEmail Optional.
+     * @param  float  $contactRevenue Optional - the monetary value this event means to you.
+     * @param  array  $contactProperties Optional - array of new contact properties.
+     */
+    public function addEvent($hubId, $eventId,
```

Comment:
Can we rename `addEvent` to `trigger` so as not to be confused with creating a new event?

and matches hubspot's language

> If you're using HubSpot Events on the Enterprise version of the HubSpot software, you can use this call to **trigger** any custom events that you define in HubSpot (under "Reports -> Events"). You can then use events that you **trigger** to create smart lists, score leads and email contacts.